README.md

```diff
@@ -1,7 +1,7 @@
 
 # Code Review Checklist: Java Concurrency
 
-Design
+<strong>Design</strong>
```

Comment:
Would it be better to use markdown syntax instead? Example:
```suggestion
**Design**
```

Reply:
Yes definitely,  we can use it. 
Thanks.